README.md

```diff
@@ -82,9 +86,13 @@ and concerns.
 * Edit.  Be sure no leftover text exists.  No tabs either.
 * If you are translating, remember to add the full Git commit hash of the English
   file that you are translating from, to the file's `EN-Revision` comment.
+* cd into the desired clone directory, e.g.
+  ```bash
+  cd en
+  ```
```

Comment:
Might make sense to say this name is hardcoded? As when you git clone you'd get ``doc-en`` as the folder name, so or a symlink is necessary or ``git clone link-to-doc-en-repo.git en`` should be done while cloning doc-en

Reply:
When cloning from git.php.net (like described further up the page), the default directory name is `en`.

Reply:
Ah, didn't know it was referencing git.php.net directly.